b3/plugins/xlrstats/__init__.py

```diff
@@ -1823,7 +1826,7 @@ def cmd_xlrinit(self, data, client, cmd=None):
         xlr_tables = []
         for attr in dir(self):
             attr_val = getattr(self, attr)
-            if attr.endswith('_table') and attr_val.startswith('xlr_'):
+            if attr.endswith('_table') and attr_val not in self.exclude_from_xlrinit:
```

Comment:
Since it's being used only here I don't feel the need of storing the tables in an attribute (and use sets instead of lists 😉). I would just do:

```
if attr.endswith('_table') and attr_val not in {'clients', 'penalties'}:
```